packages/synth.queues/src/services/kafka.queue.service.ts

```diff
@@ -0,0 +1,118 @@
+import { PluginManager } from '@applicature/synth.plugin-manager';
+import * as kafka from 'node-rdkafka';
+import * as logger from 'winston';
+import { KafkaConfig, Message, Queue } from '../types';
+import { QueueService } from './queue.service';
+export class KafkaQueueService extends QueueService {
+    private producerReady: any;
+    private consumerReady: any;
```

Comment:
you should init consumer in `init` method, and use an instance of KafkaConsumer, instead of Promise< KafkaConsumer >